README.md

```diff
@@ -1,8 +1,18 @@
 # User Management Plugin
 
+User Management > 2.0.0 supports Grails 3.3.x
+User Management < 1.0.2 supports Grails 3.2.x (Tested upto 3.2.5)
```

Comment:
`User Management >= 2.0.0 supports Grails 3.3.x`

`User Management <= 1.0.2 supports Grails 3.2.x `

Reply:
Also, For 3.3.x you can add a statement in a bracket (tested up to 3.3.5)